.github/workflows/publish_private_website.yml

```diff
@@ -25,6 +25,9 @@ jobs:
     steps:
     - uses: 'actions/checkout@v4'
 
+    - name: Set up Cloud SDK
+      uses: google-github-actions/setup-gcloud@v2    
+
     - name: Authenticate with Google Cloud
       uses: 'google-github-actions/auth@v2.1.3'
       with:
```

Comment:
what about with this?

```
 with:
          credentials_json: "${{ secrets.ARTIFACT_REGISTRY_KEY }}"
```

Reply:
this is an action that someone else will call, and they will provide the secrets.